src/main/webapp/.yarnrc.yml

```diff
@@ -1 +1 @@
-yarnPath: ".yarn/releases/yarn-berry.cjs"
```

Comment:
Shouldn't you do this off development?

Reply:
Ended up doing a PR into dev and merging into this branch